src/scripts/main.js

```diff
@@ -1,3 +1,44 @@
 'use strict';
 
-// write code here
+// select all <li> that are direct child of <ul>
+const list = document.querySelectorAll('ul > li');
+
+function sortList(itemsList) {
+  // Return new sorted array
+  // originaly list = [li, li, li], each <li> is an object.
+  // Each <li> object has dataset.
+  // Each dataset has the salary key which stores string (salary value)
```

Comment:
Remove comment. Try to write obvious code that explains all we need without comments)